packages/esbuild-plugin/src/plugins.ts

```diff
@@ -7,15 +7,10 @@
 import { PluginBuild } from 'esbuild';
 import { performance } from 'perf_hooks';
 
-import { TimingsMap, Timing, Value } from '../types';
-import { getContext, formatModuleName } from '../helpers';
+import { TimingsMap, Timing, Value } from '@datadog/build-plugins-core/types';
+import { getContext, formatModuleName } from '@datadog/build-plugins-core/helpers';
 
-enum FN_TO_WRAP {
-    START = 'onStart',
-    LOAD = 'onLoad',
-    RESOLVE = 'onResolve',
-    END = 'onEnd',
-}
+const FN_TO_WRAP = ['onStart', 'onLoad', 'onResolve', 'onEnd'];
```

Comment:
If you want the type of `FN_TO_WRAP` to not be `string[]` but constant, and not have to do `fn as keyof PluginBuild`, you can just do:
```suggestion
const FN_TO_WRAP = ['onStart', 'onLoad', 'onResolve', 'onEnd'] as const;
```